charts/customer-center/values.yaml

```
@@ -0,0 +1,315 @@
+    successThreshold: 1
+  readinessProbe:
+    # backend.readinessProbe.enabled -- Enable readiness probe on backend
+    enabled: false
+    # backend.readinessProbe.path -- Path to probe
+    path: /api/v1/health
+    # backend.readinessProbe.initialDelaySeconds -- Number of seconds after the container has started before readiness probe is initiated
+    initialDelaySeconds: 30
+    # backend.readinessProbe.periodSeconds -- How often (in seconds) to perform the probe
+    periodSeconds: 10
+    # backend.readinessProbe.timeoutSeconds -- Number of seconds after which the probe times out
+    timeoutSeconds: 5
+    # backend.readinessProbe.failureThreshold -- Number of tries to perform the probe
+    failureThreshold: 6
+    # backend.readinessProbe.successThreshold -- Minimum consecutive successes for the probe to be considered successful after having failed
+    successThreshold: 1
+  settings:
+    # settings for backend
+    # backend.settings.nodeEnv -- Environment for NodeJS (can be development, testing, staging, production)
+    nodeENV: production
+    # backend.settings.debug -- Verbosity of logging
+    debug: app:info,app:warn,app:error
+    # backend.settings.appName -- Application name
+    appName: Customer Center
+    # backend.settings.appHost -- Application host name
+    appHost: my.example.com
+    # backend.settings.keycloak -- Keycloak settings
+    keycloak:
+      # backend.settings.keycloak.host -- Keycloak host
+      host: https://sso.example.com
+      # backend.settings.keycloak.tokenPath -- Keycloak token path
+      tokenPath: /auth/realms/example/protocol/openid-connect/token
+      # backend.settings.keycloak.clientID -- Keycloak client ID
+      clientID: this-is-my-client-id
+      # backend.settings.keycloak.clientSecret -- Keycloak client secret
+      clientSecret: this-is-my-client-secret
+    # backend.settings.timed -- Timed settings
+    timed:
+      # backend.settings.timed.host -- Timed host
+      host: https://timed.example.com
+      # backend.settings.timed.prefix -- Timed prefix
+      prefix: /api/v1
+    # backend.settings.smtp -- SMTP settings
+    smtp:
+      # backend.settings.stmp.host -- SMTP server host
+      host: mail.example.com
+      # backend.settings.smtp.from -- Default email address to use for various responses
+      from: customer-center@example.com
+      # backend.settings.stmp.to -- Default destination email address
+      to: administration@example.com
+      # backend.settings.stmp.secure -- If true the connection will use TLS when connetion to server
+      secure: true
+      # backend.settings.stmp.requireTLS -- If this is true and secure is false then Nodemailer tries to use STARTTLS even if the server does not advertise support for it. If the connection can not be encrypted then message is not sent
+      requireTLS: true
+    # backend.settings.auth -- Authentication settings
+    auth:
```

To get rid of the JSON docs block for `backend.settings.auth` you can just leave out the description of the top-level key to suppress the whole line.
```suggestion
    auth:
```

---

Thanks for the help about removing those huge JSON blocks.